sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/dynamicconfig/ConfigClient.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sermant.implement.service.dynamicconfig;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Configuration Center Client Interface
+ *
+ * @author zhp
+ * @since 2024-05-16
+ */
+public interface ConfigClient {
+    /**
+     * Get configuration
+     *
+     * @param key configuration key
+     * @param group configuration group
+     * @return configuration content
+     */
+    String getConfig(String key, String group);
+
+    /**
+     * Get configurations
+     *
+     * @param key configuration key
```

Comment:
configuration key for fuzzy search. configuration group for fuzzy search

Reply:
The parameter of the matching mode is added.